tests/instancetype/instancetype.go

```diff
@@ -1743,7 +1760,7 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 							Spec: virtv1.VirtualMachineInstanceSpec{
 								Domain: virtv1.DomainSpec{
 									Memory: &virtv1.Memory{
-										Guest: resource.NewQuantity(2*1024*1024*1024, resource.BinarySI),
+										Guest: resource.NewQuantity(memorySize2GB, resource.BinarySI),
```

Comment:
Please consider using https://pkg.go.dev/k8s.io/apimachinery/pkg/api/resource#MustParse:
```suggestion
										Guest: resource.MustParse("2Gi"),
```
Same in other places defining memory quantity.